sample/wikifile.py

```diff
@@ -0,0 +1,76 @@
+import requests
+from pprint import pprint
+import json
+
+with open('metadata.json') as data_file:
+    data = json.load(data_file)
+
+
+class WikiFile:
+    def __init__(self, **reference_to_pages):
+        self.base_url = "https://en.wikipedia.org/w/api.php"
+
+        self.payload = {
+            'action': 'query',
+            'format': 'json',
+        }
+        self.headers = {
+                'User-Agent': data["name"] + ' (' + data["website"] + ')'
+        }
+
+        self._parse_kwargs(**reference_to_pages)
+
+    def _parse_kwargs(self, **kwparams):
+        if 'pageids' in kwparams:
+            self.payload['pageids'] = '|'.join(str(ID) for ID in list(kwparams['pageids']))
+        elif 'titles' in kwparams:
+            self.payload['titles'] = '|'.join(str(ID) for ID in list(kwparams['titles']))
```

Comment:
All `pageids` won't be `-1`, please check this [response](https://en.wikipedia.org/w/api.php?action=query&titles=Image:1995.jpg%7CImage:Welcome.gif&prop=duplicatefiles).

Nonetheless, I have changed key to be `title`.